src/lib/components/Channel/Chat/DrawerRestream.svelte

```diff
@@ -113,6 +113,22 @@
 		const rtmps = await getLiveInput()
 		await sendOutputs({ liveInputUid: rtmps.rtmps.uid })
 	}
+
+	const getTwitchStreamLink = async () => {
+		const linkRes = await get(`twitch/stream/link?channelId=${$page.params.channelId}`, auth)
+
+		payload.streamKey = linkRes.streamKey
+		selected = 'twitch'
+	}
+
+	const getYoutubeStreamLink = async () => {
+		const linkRes = await get(`youtube/stream/link?channelId=${$page.params.channelId}`, auth)
+
+		;(payload.url = linkRes.streamaddress),
+			(payload.streamKey = linkRes.streamKey),
+			(selected = 'youtube')
+			console.log(selected)
+	}
```

Comment:
What does this code do?

Reply:
This is basically to show and hide field based on selected platform